From 1c7b99ae7e89f50b17419ae07039adbe34f45b45 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Thu, 25 Jan 2024 17:36:09 +0100
Subject: [PATCH 474/954] refactor(stm32mp2): pwr: remove useless structure
 fields

Cleanup useless structure fields in order to
reduce code size.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 3a18dfefc81959de5c2729b889d68d5ed5d630d2 refactor(stm32mp2): pwr: remove useless structure fields]
Change-Id: If89f8e2d4048b52a765a9fb42567c36f91e17650
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/354523
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 plat/st/stm32mp2/stm32mp2_pwr.c | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/plat/st/stm32mp2/stm32mp2_pwr.c b/plat/st/stm32mp2/stm32mp2_pwr.c
index 457205ba2a..4ca47a737a 100644
--- a/plat/st/stm32mp2/stm32mp2_pwr.c
+++ b/plat/st/stm32mp2/stm32mp2_pwr.c
@@ -18,8 +18,6 @@
 struct pwr_regu {
 	const char *node_name;
 	uint32_t enable_reg;
-	uint32_t enable_mask;
-	uint32_t ready_mask;
 	uint32_t valid_mask;
 	uint32_t vrsel_mask;
 	int comp_idx;
@@ -39,8 +37,6 @@ static const struct pwr_regu pwr_regulators[] = {
 	 [IOD_VDDIO1] = {
 		.node_name = "vddio1",
 		.enable_reg = PWR_CR8,
-		.enable_mask = PWR_CR8_VDDIO1VMEN,
-		.ready_mask = PWR_CR8_VDDIO1RDY,
 		.valid_mask = PWR_CR8_VDDIO1SV,
 		.vrsel_mask = PWR_CR8_VDDIO1VRSEL,
 		.comp_idx = SYSFG_VDDIO1_ID,
@@ -49,8 +45,6 @@ static const struct pwr_regu pwr_regulators[] = {
 	 [IOD_VDDIO2] = {
 		.node_name = "vddio2",
 		.enable_reg = PWR_CR7,
-		.enable_mask = PWR_CR7_VDDIO2VMEN,
-		.ready_mask = PWR_CR7_VDDIO2RDY,
 		.valid_mask = PWR_CR7_VDDIO2SV,
 		.vrsel_mask = PWR_CR7_VDDIO2VRSEL,
 		.comp_idx = SYSFG_VDDIO2_ID,
@@ -59,8 +53,6 @@ static const struct pwr_regu pwr_regulators[] = {
 	 [IOD_VDDIO3] = {
 		.node_name = "vddio3",
 		.enable_reg = PWR_CR1,
-		.enable_mask = PWR_CR1_VDDIO3VMEN,
-		.ready_mask = PWR_CR1_VDDIO3RDY,
 		.valid_mask = PWR_CR1_VDDIO3SV,
 		.vrsel_mask = PWR_CR1_VDDIO3VRSEL,
 		.comp_idx = SYSFG_VDDIO3_ID,
@@ -69,8 +61,6 @@ static const struct pwr_regu pwr_regulators[] = {
 	 [IOD_VDDIO4] = {
 		.node_name = "vddio4",
 		.enable_reg = PWR_CR1,
-		.enable_mask = PWR_CR1_VDDIO4VMEN,
-		.ready_mask = PWR_CR1_VDDIO4RDY,
 		.valid_mask = PWR_CR1_VDDIO4SV,
 		.vrsel_mask = PWR_CR1_VDDIO4VRSEL,
 		.comp_idx = SYSFG_VDDIO4_ID,
-- 
2.53.0

